content/wissen/test4/index.md

```diff
@@ -0,0 +1,32 @@
+---
+draft: true
+
+title: "Test page 4"
+description: "How to set up a new branch, make changes and make a pullrequest"
+topics: ["test", "abc"]
+authors:
+  - "fackerl"
+series: ["Strommärkte"]
+series_order: 2
+weight: 2
+---
+
+# Test
```

Comment:
Please check: If the page that get's rendered already has a "large" title `Test page 4` then I would go for using `##` as the first allowed header level and not using an `h1` (`#`) at all - in all articles.

If that is as I remember, then please open an issue regarding that we should add stuff like that (and maybe also the markdownlint stuff) into the "writing" section of the README. But that can be a separate PR, no need to include that here.

Reply:
In der preview wird bei mir kein großer Titel "Test page 4" angezeigt. Warum ist das trotzdem ein Problem, die erste Überschrift größer zu machen? Oder meinst du einfach, dass wir generell keine großen # headers verwenden?